src/Symfony/Component/HttpFoundation/Tests/JsonResponseTest.php

```diff
@@ -16,6 +16,15 @@
 
 class JsonResponseTest extends TestCase
 {
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (!defined('HHVM_VERSION')) {
+            $this->iniSet('serialize_precision', 14);
```

Comment:
Shouldn't we revert to the current value after the tests?

Reply:
that's exactly why the test is using `$this->iniSet` and not `ini_set`. PHPunit already has this feature of configuring a ini setting and reverting it at the end of the test

Reply:
the iniSet helper does it already!